nservicebus/messaging/routing.md

```diff
@@ -119,17 +119,17 @@ When necessary (e.g. for integration with legacy systems) the static routing to
 snippet:Routing-StaticRoutes-Address
 
 
-#### Dynamic routes
+#### Fallback route
```

Comment:
note: fallback route and dynamic routes will most likely be removed in the next beta